src/modules/localStorageFunctions.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
```

Comment:
- [OPTIONAL] It is always a best practice to make sure all linters have been fixed manually instead of disabling them, linter is excellent for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. Kindly fix this

Reply:
Done ✔️